### DISCUSSION TITLE
Gurobi parameter setting 'outputflag' and 'OutputFlag': same names twice

### DISCUSSION THREAD

**Discussion by cchen000** [yalmip/YALMIP#1373] [Possible bug or missing functionality]
Hi Johan,

I am a Matlab and Gurobi user. Recently, I am learning how to control the output option by parameter settings. This is not a complex task, and I successfully made it by following the way how YALMIP does. Thanks for your project.

And I also find that some name typos (possibly) of optimize options in current project: `outputflag` and `OutputFlag`.

-  For the official [Gurobi website](https://www.gurobi.com/documentation/9.0/refman/parameters.html#sec:Parameters), there is a field `OutputFlag`. 
-  For current project, the file `solvers/yalmip2gurobi.m` sets the field `outputflag`. So, this creates a new field very similar to `OutputFlag`.

So, are they the same? Two different fields both exist before calling gurobi function.

If so, from my side, these fields can be unified with a field `OutputFlag` as the official website does. The same goes for another field `nonconvex` and `NonConvex`. Until now, I do not find any numerical error related to `outputflag` and `OutputFlag`. This is just a suggestion to regularize the code.

--
cc

**Comment by johanlofberg** [comment]
Yes, verbosity is controlled in yalmip viathe verbose option, which then alters solver specific settings (outputflag in the case of gurobi), and nonconvex is also controlled by yalmip

**Reply by johanlofberg** [reply]
Now I see your concern. does not seem to cause any issues but I'll clean that up
